lib/huginn_scheduler.rb

```diff
@@ -169,6 +169,7 @@ def cleanup_expired_events!
   end
 
   def cleanup_failed_jobs!
+    return if Rails.configuration.active_job.queue_adapter != :delayed_job
```

Comment:
Is there an equivalent for sidekiq, or does it never keep them around?

Reply:
It is now using the build in failure tracking of sidekiq